bin/start.sh

```diff
@@ -13,5 +13,11 @@ php /data/bin/install.php
 echo Applying configuration file security
 chmod 644 /data/upload/include/ost-config.php
 
+mkdir /run/nginx
```

Comment:
My container fails to restart probably due to this line. Error log says: `mkdir: can't create directory '/run/nginx': File exists`
Same issue will rise probably with the `mkdir /var/log/php` line below as well.